app/styles/partials/overrides.scss

```diff
@@ -18,3 +18,7 @@ body.dimmable.undetached.dimmed {
     position: static;
   }
 }
+
+.ui.grid.grid {
```

Comment:
Please do not do global overrides like this unless there is a specific reason. This would override grids throughout the app.